doc/installation.md

```diff
@@ -88,3 +88,28 @@ If you are using Spacemacs, you can easily install org-roam by creating a simple
 ```
 
 Next, append `org-roam` to the `dotspacemacs-configuration-layers` list in your `.spacemacs` configuration file. Reload (`SPC f e R`) or restart Emacs to load `org-roam`. It's functions are available under the prefix `SPC a r` and `, r` when visiting an org-mode buffer. 
+
+## Doom Emacs
+
+If you are using [Doom Emacs](https://github.com/hlissner/doom-emacs), configure packages as explained in the [getting started](https://github.com/hlissner/doom-emacs/blob/develop/docs/getting_started.org#configuring-packages) guide. 
+
+Declare org-roam as a package in your `~/.doom.d/packages.el`:
+
+```elisp
+;; ~/.doom.d/packages.el
+
+(package! org-roam
+  :recipe (:host github :repo "jethrokuan/org-roam" :branch "develop"))
+```
+
+Subsequently, in your `~/.doom.d/config.el` file, configure org-roam:
+
+```elisp
+;; ~/.doom.d/config.el
+(use-package! org-roam
```

Comment:
Do we need a `:after org` here? 

/cc @hlissner

Reply:
No. If you wish to have the bindings available before Org is loaded, then you can leave out `:after org`.

Reply:
If we're binding leader keys, we could bind them early and activate org-roam-mode when any of those commands autoload the package. i.e.

```elisp
(use-package! org-roam
  :commands (org-roam-insert org-roam-find-file org-roam)
  :init 
  (setq org-roam-directory "/path/to/org-files/")
  (map! :leader 
        :prefix "n"
        :desc "Org-Roam-Insert" "i" #'org-roam-insert
        :desc "Org-Roam-Find"   "/" #'org-roam-find-file
        :desc "Org-Roam-Buffer" "r" #'org-roam)
  :config
  (org-roam-mode +1))
```

What do you think?